Alexis-Vacilli/myBrand-api
# Ch server setup

#### What does this PR do?
[what does this PR do?]
> This PR is about setting up the server 
#### Description of Task to be completed
[the tasks that you completed on the feature]
> The task completed is about setting up the server 
#### How should this be manually tested?
> Clone this repo https://github.com/Alexis-Vacilli/myBrand-api.git
> npm install: To install all the dependencies
> npm run cover: To run tests and see the coverage
#### Any background context you want to provide?
> None 
#### What are the relevant Trello story?
> https://trello.com/c/JX2Y8FpF
#### Screenshots (if appropriate)
>N/A
